internal/oci/annotations.go

```diff
@@ -0,0 +1,208 @@
+	AnnotationBootFilesRootPath = "io.microsoft.virtualmachine.lcow.bootfilesrootpath"
+
+	// AnnotationKernelDirectBoot indicates that we should skip UEFI and boot directly to `kernel`
+	AnnotationKernelDirectBoot = "io.microsoft.virtualmachine.lcow.kerneldirectboot"
+
+	// AnnotationVPCIEnabled indicates that pci support should be enabled for the LCOW UVM
+	AnnotationVPCIEnabled = "io.microsoft.virtualmachine.lcow.vpcienabled"
+
+	// AnnotationVPMemNoMultiMapping indicates that we should disable LCOW vpmem layer multi mapping
+	AnnotationVPMemNoMultiMapping = "io.microsoft.virtualmachine.lcow.vpmem.nomultimapping"
+
+	// AnnotationStorageQoSBandwidthMaximum indicates the maximum number of bytes per second. If `0`
+	// will default to the platform default.
+	AnnotationStorageQoSBandwidthMaximum = "io.microsoft.virtualmachine.storageqos.bandwidthmaximum"
+
+	// AnnotationStorageQoSIopsMaximum indicates the maximum number of Iops. If `0` will
+	// default to the platform default.
+	AnnotationStorageQoSIopsMaximum = "io.microsoft.virtualmachine.storageqos.iopsmaximum"
+
+	// AnnotationFullyPhysicallyBacked indicates that the UVM should use physically backed memory only,
+	// including for additional devices added later.
+	AnnotationFullyPhysicallyBacked = "io.microsoft.virtualmachine.fullyphysicallybacked"
+
+	// AnnotationDisableCompartmentNamespace sets whether to disable namespacing the network compartment in the UVM
+	// for WCOW.
+	AnnotationDisableCompartmentNamespace = "io.microsoft.virtualmachine.disablecompartmentnamespace"
+
+	// AnnotationVSMBNoDirectMap specifies that no direct mapping should be used for any VSMBs added to the UVM
+	AnnotationVSMBNoDirectMap = "io.microsoft.virtualmachine.wcow.virtualSMB.nodirectmap"
+
+	// annotation used to specify the cpugroup ID that a UVM should be assigned to
+	AnnotationCPUGroupID = "io.microsoft.virtualmachine.cpugroup.id"
+
+	// SaveAsTemplate annotation must be used with a pod & container creation request.
+	// If this annotation is present in the request then it will save the UVM (pod)
+	// and the container(s) inside it as a template. However, this also means that this
+	// pod and the containers inside this pod will permananetly stay in the
+	// paused/templated state and can not be resumed again.
+	AnnotationSaveAsTemplate = "io.microsoft.virtualmachine.saveastemplate"
+
+	// This annotation should be used when creating a pod or a container from a template.
+	// When creating a pod from a template use the ID of the templated pod as the
+	// TemplateID and when creating a container use the ID of the templated container as
+	// the TemplateID. It is the client's responsibility to make sure that the sandbox
+	// within which a cloned container needs to be created must also be created from the
+	// same template.
+	AnnotationTemplateID = "io.microsoft.virtualmachine.templateid"
+
+	// AnnotationNetworkConfigProxy holds the address of the network config proxy service.
```

Comment:
Can add "if this is present, network setup will attempt to be done through ncproxy"

Reply:
done